scripts/dotnet-cli-build/CompileTargets.cs

```diff
@@ -645,11 +640,6 @@ private static BuildTargetResult CompileCliSdk(BuildTargetContext c, DotNetCli d
 
             CrossgenUtil.CrossgenDirectory(c, outputDir);
 
-            // Generate .version file
-            var version = buildVersion.NuGetVersion;
-            var content = $@"{c.BuildContext["CommitHash"]}{Environment.NewLine}{version}{Environment.NewLine}";
-            File.WriteAllText(Path.Combine(outputDir, ".version"), content);
```

Comment:
Does this mean that the SDK directory won't contain a .version file any more?  We have code that depends on that - specifically: https://github.com/dotnet/cli/blob/rel/1.0.0/src/Microsoft.DotNet.Cli.Utils/DotnetFiles.cs#L16